jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -64,98 +62,26 @@ def __init__(self, *args, **kwargs):
-
-            # Extract first dataset from visible layers and use this for coordinates - the choice
-            # of dataset shouldn't matter if the datasets are linked correctly
-            image = active_layer.layer
-            self.label_mouseover.icon = self.jdaviz_app.state.layer_icons.get(active_layer.layer.label)  # noqa
-
-            # Extract data coordinates - these are pixels in the reference image
-            x = data['domain']['x']
-            y = data['domain']['y']
-
-            if x is None or y is None:  # Out of bounds
-                self.label_mouseover.pixel = ""
-                self.label_mouseover.reset_coords_display()
-                self.label_mouseover.value = ""
-                return
-
-            maxsize = int(np.ceil(np.log10(np.max(image.shape)))) + 3
-            fmt = 'x={0:0' + str(maxsize) + '.1f} y={1:0' + str(maxsize) + '.1f}'
-            x, y, coords_status, (unreliable_world, unreliable_pixel) = self._get_real_xy(image, x, y)  # noqa
-            self.label_mouseover.pixel = (fmt.format(x, y))
-
-            if coords_status:
-                try:
-                    coo = image.coords.pixel_to_world(x, y).icrs
-                except Exception:  # WCS might not be celestial
-                    self.label_mouseover.reset_coords_display()
-                else:
-                    self.label_mouseover.set_coords(coo,
-                                                    unreliable_world=unreliable_world,
-                                                    unreliable_pixel=unreliable_pixel)  # noqa
-            else:
-                self.label_mouseover.reset_coords_display()
-
-            # Extract data values at this position.
-            # TODO: for now we just use the first visible layer but we should think
-            # of how to display values when multiple datasets are present.
-            if (x > -0.5 and y > -0.5
-                    and x < image.shape[1] - 0.5 and y < image.shape[0] - 0.5
-                    and hasattr(active_layer, 'attribute')):
-                attribute = active_layer.attribute
-                value = image.get_data(attribute)[int(round(y)), int(round(x))]
-                unit = image.get_component(attribute).units
-                self.label_mouseover.value = f'{value:+10.5e} {unit}'
-            else:
-                self.label_mouseover.value = ''
-
-        elif data['event'] == 'mouseleave' or data['event'] == 'mouseenter':
-
-            self.label_mouseover.pixel = ""
-            self.label_mouseover.reset_coords_display()
-            self.label_mouseover.value = ""
-
-        elif data['event'] == 'keydown':
+        if data['event'] == 'keydown':
             key_pressed = data['key']
 
             if key_pressed in ('b', 'B'):
                 self.blink_once(reversed=key_pressed=='B')  # noqa: E225
 
-                # Also update the coordinates display.
-                data['event'] = 'mousemove'
```

Comment:
Need to check if blink without moving cursor still correct.

Reply:
(same as above - this should work, but please feel free to test and confirm or let me know if any case doesn't act as expected)

Reply:
Looks like it works to me.